test/performance/latency/LatencyTestPublisher.hpp

```diff
@@ -135,6 +140,10 @@ class LatencyTestPublisher
     eprosima::fastdds::dds::DataReader* data_reader_;
     eprosima::fastdds::dds::DataReader* command_reader_;
 
+    /* QoS Profiles */
+    eprosima::fastdds::dds::DataReaderQos dr_qos;
+    eprosima::fastdds::dds::DataWriterQos dw_qos;
```

Comment:
```suggestion
    eprosima::fastdds::dds::DataReaderQos dr_qos_;
    eprosima::fastdds::dds::DataWriterQos dw_qos_;
```

Reply:
Done in 759afcbf9cb2c567b30b3b3475617648754d5895